private-templates-service/client/src/components/Dashboard/RecentlyViewed/styled.tsx

```diff
@@ -42,7 +42,7 @@ export const RecentlyViewedBody = styled.div`
   justify-content: flex-start;
 `;
 
-export const RecentlyViewedItem = styled.h6`
+export const RecentlyViewedItem = styled.div`
```

Comment:
this really didn't make a difference in the font-size?))

Reply:
Nope, it stayed the same